r/tools/linuxlibs.R

```diff
@@ -315,8 +315,13 @@ build_libarrow <- function(src_dir, dst_dir) {
 }
 
 ensure_cmake <- function() {
-  cmake <- Sys.which("cmake")
-  if (!nzchar(cmake) || cmake_version() < 3.2) {
+  cmake <- find_cmake(c(
+    Sys.getenv("CMAKE"),
+    Sys.which("cmake"),
+    Sys.which("cmake3")
+  ))
+
+  if (is.null(cmake)) {
     # If not found, download it
     cat("**** cmake\n")
     CMAKE_VERSION <- Sys.getenv("CMAKE_VERSION", "3.16.2")
```

Comment:
You probably want to bump that version: latest is either 3.18.1, 3.17.4 or 3.16.8.

Reply:
Right, will do. Can't remember where I got that version number from before--probably one of our CI jobs.